### PR TITLE
Fix varchar(191) by replacing with 255 #135

### LINKED ISSUE #135: Fix varchar(191) by replacing with 255
https://www.grouparoo.com/blog/varchar-191

https://stackoverflow.com/questions/43832166/laravel-5-4-specified-key-was-too-long-why-the-number-191

### PR DESCRIPTION
**Pull request by RMartinOscar**
Edited Filament, Models, mysql-schema and made new migration to fix https://github.com/pelican-dev/panel/issues/135

### CODE REVIEW

File: app/Providers/AppServiceProvider.php [hunk 1]
```diff
@@ -29,7 +29,7 @@ class AppServiceProvider extends ServiceProvider
      */
     public function boot(): void
     {
-        Schema::defaultStringLength(191);
+        Schema::defaultStringLength(255);
```
Comment (lancepioch) [inline app/Providers/AppServiceProvider.php:32]: Don't need this anymore!

File: database/migrations/2024_06_11_220722_update_field_length.php [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('activity_log_subjects', function (Blueprint $table) {
+            $table->string('subject_type', 255)->change();
```
Comment (lancepioch) [inline database/migrations/2024_06_11_220722_update_field_length.php:15]: Omit 255 as the parameter for each of them, it's the default!

**Review by lancepioch — CHANGES_REQUESTED** [review]

File: database/migrations/2024_06_11_220722_update_field_length.php (hunk 2 above)
Comment (RMartinOscar) [reply database/migrations/2024_06_11_220722_update_field_length.php:15]: I tried without specifying it won't overwrite 191.

File: app/Providers/AppServiceProvider.php (hunk 1 above)
Comment (RMartinOscar) [reply app/Providers/AppServiceProvider.php]: Removed aswell as the useless import

**Review by Boy132 — APPROVED** [review]
LGTM

**Review by lancepioch — APPROVED** [review]

**Pull request merged**